HermesProxy/World/Server/PacketHandlers/ItemHandler.cs

```diff
@@ -196,5 +196,16 @@ void HandleCancelTempEnchantment(CancelTempEnchantment cancel)
             packet.WriteUInt32(cancel.EnchantmentSlot);
             SendPacketToServer(packet);
         }
+
+        [PacketHandler(Opcode.CMSG_WRAP_ITEM)]
+        void HandleWrapItem(WrapItem item)
+        {
+            WorldPacket packet = new WorldPacket(Opcode.CMSG_WRAP_ITEM);
+            packet.WriteUInt8(item.GiftBag);
+            packet.WriteUInt8(item.GiftSlot);
+            packet.WriteUInt8(item.ItemBag);
+            packet.WriteUInt8(item.ItemSlot);
```

Comment:
Don't you need to do conversion for the slots? Cause slots are different between versions.

Reply:
They're UInt8 on both CMaNGOS TBC and VMaNGOS
Order is also the same on both.

Reply:
I think he means that the client structure might be different + that the slots itself might have a different meaning in the modern client.

Reply:
the error responses are all off by one it seems, tho.

Reply:
`CantTradeGold = 29, // Gold May Only Be Offered By One Trader.`
doesn't exist in CMaNGOS

Reply:
> I think he means that the client structure might be different + that the slots itself might have a different meaning in the modern client.

makes sense!
I tested it with the 2.5.2 Client at least. I'll test again with 1.14, but I'm almost confident that they didn't change this particular opcode

Reply:
Checked with the 1.14 client, works identically over there.

Reply:
The inventory slots are different in every version. So if you send them as is, you will target wrong item. It will only work on items in the main bag without conversion. Please refer to these definitons.

https://github.com/WowLegacyCore/HermesProxy/blob/master/HermesProxy/World/Enums/PlayerDefines.cs#L830